.azure-pipelines/linux-conda-CI.yml

```diff
@@ -60,16 +60,20 @@ jobs:
 
   - script: |
       test '$(python.version)' == '3.7' && apt-get install protobuf-compiler libprotoc-dev
-      conda config --set always_yes yes --set changeps1 no      
+      conda config --set always_yes yes --set changeps1 no
       conda install -c conda-forge numpy$(numpy.version)
       conda install protobuf
-      python -m pip install --upgrade pip      
+      python -m pip install --upgrade pip
     displayName: 'Install environment'
 
   - script: |
       pip install flake8
     displayName: 'install flake8'
 
+  - script: |
```

Comment:
I removed it for linux and let it for Windows. That way, a build does not get stuck because of style. It usually saves one commit.

Reply:
Makes sense! I'll revert my change.